doc/tutorials/nvmx_virtualization.rst

```diff
@@ -0,0 +1,332 @@
+        :align: center
+
+        Service VM legacy and console vUART settings
+
+- Build with the XML configuration, referring to :ref:`getting-started-building`.
+
+
+Prepare for Service VM Kernel and rootfs
+****************************************
+
+The service VM can run Ubuntu or other Linux distributions.
+Instructions on how to boot Ubuntu as the Service VM can be found in
+:ref:`rt_industry_ubuntu_setup`.
+
+The Service VM kernel needs to be built from the ``acrn-kernel`` repo, and some changes
+to the kernel ``.config`` are needed.
+
+Instructions on how to build and install the Service VM kernel can be found
+in :ref:`Build and Install the ACRN Kernel <build-and-install-ACRN-kernel>`.
+
+Here is the quick start of how to modify and build the kernel:
+
+.. code-block:: none
+
+   git clone https://github.com/projectacrn/acrn-kernel
+   cd acrn-kernel
+   git checkout master
+   cp kernel_config_uefi_sos .config
+   make olddefconfig
+
+The following configuration entries are needed to launch nested
+guests on the Service VM:
+
+.. code-block:: none
+
+   CONFIG_KVM=y
+   CONFIG_KVM_INTEL=y
+   CONFIG_ACRN_GUEST=y
+
+This setting is also needed if you want to use virtio block device as root filesystem
+for the guest VMs:
+
+.. code-block:: none
+
+   CONFIG_VIRTIO_BLK=y
+
+After the configuration modifications, build and install the kernel.
+
+.. code-block:: none
+
+   make all
+
+
+Launch a Nested Guest VM
+************************
+
+Create an Ubuntu KVM Image
+==========================
+
+Refer to :ref:`Build the Ubuntu KVM Image <build-the-ubuntu-kvm-image>`
+on how to create an Ubuntu KVM image as the nested guest VM's root filesystem.
+There is no particular requirement for this image, e.g., it could be of either
+qcow2 or raw format.
+
+Prepare for Launch Scripts
+==========================
+
+Install QEMU on the Service VM that will launch the nested guest VM:
+
+.. code-block:: none
+
+   sudo apt-get install qemu-kvm qemu virt-manager virt-viewer libvirt-bin
+
+The following is a simple example for the script to launch a nested guest VM.
+
+.. important:: The ``-cpu host`` option is needed to launch a nested guest VM, and ``-nographics``
+   is required to run nested guest VMs reliably.
+
+Besides this, there is no particular requirements for the launch script.
+You can prepare the script just like the one you use to launch a VM
+on native Linux.
+
+For example, other than ``-hda``, you can use the following option to launch
+a virtio block based RAW image: ``-drive format=raw,file=/root/ubuntu-20.04.img,if=virtio``
+
+Use the following option to enable Ethernet on the guest VM:
+``-netdev tap,id=net0 -device virtio-net-pci,netdev=net0,mac=a6:cd:47:5f:20:dc``
+
+.. code-block:: bash
+   :emphasize-lines: 2-4
+
+   sudo qemu-system-x86_64 \
+     -enable-kvm \
+     -cpu host \
+     -nographic \
+     -m 2G -smp 2 -hda /root/ubuntu-20.04.qcow2 \
+     -net nic,macaddr=00:16:3d:60:0a:80 -net tap,script=/etc/qemu-ifup
```

Comment:
Does that automatically attach the VM to the `acrn-br0` bridge?

Reply:
No acrn-br0 is needed her. User can check the generic QEMU documentation on how to launch nested guests.

Reply:
I understand it's not strictly needed, but what happens if it is present (because of other User VMs being run on the system for example). Are we re-using it? Can ACRN-launched User VMs communicate with QEMU Guest VMs if both are running on the same system?